container-storage-setup.spec

```diff
@@ -26,21 +26,11 @@ as the root logical volume and partition table.
 %install
 %make_install
 
-%post
-%systemd_post %{name}.service
-
-%preun
-%systemd_preun %{name}.service
-
-%postun
-%systemd_postun %{name}.service
-
 %files
-%{_unitdir}/docker-storage-setup.service
```

Comment:
We are shipping a container-storage-setup package it does not include any docker services. Those will ship in the docker package.